xtensor-blasConfig.cmake.in

```diff
@@ -34,3 +34,15 @@ set(${PN}_INCLUDE_DIRS "${TEMP_XTENSOR_BLAS_INCLUDE_DIR}")
 unset(TEMP_XTENSOR_BLAS_INCLUDE_DIR)
 set(${PN}_LIBRARY "")
 check_required_components(${PN})
+
+if (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} VERSION_GREATER_EQUAL 3.11)
+  # Microsoft compiler
+  if(NOT CMAKE_${COMPILER_LANGUAGE}_COMPILER_IS_MSVC)
+    # gcc, clang, ...
+    include(CheckCXXCompilerFlag)
+    CHECK_CXX_COMPILER_FLAG(-march=native arch_native_supported)
+    if(arch_native_supported)
+      target_compile_options(xtensor_blas INTERFACE -march=native)
+    endif()
+  endif()
+endif()
```

Comment:
Since xtensor-blas is a header-only library, we want to avoid exporting this flag, and let the user choose how he wants to build. Consider a binary taht you want to distribute through a package manager, you would probably not set this flag to be compatible with as many compatible platforms as possible, not only those compatible with the buld machine.

Reply:
Thanks @JohanMabille that makes sense, is the correct fix, in this case just to delete the flag, in:

https://github.com/xtensor-stack/xtensor-blas/blob/33ffcf271f6e8a2b94b1de9d41fab9ff2b810b1e/test/CMakeLists.txt#L60

and the other places, or do you suggest something else?

Reply:
I think it makes sense to keep it in the test and the benchmarks folders, since they are supposed to be built locally and not districbuted as binary. My remar was only for the config.cmake.in file.

Reply:
Aha, I see got it.